akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -63,7 +63,7 @@ class Slf4jLogger extends Actor with SLF4JLogging with RequiresMessageQueue[Logg
   val mdcAkkaUidAttributeName = "akkaUid"
 
   private def akkaAddress = context.system.asInstanceOf[ExtendedActorSystem].provider.addressString
-  private val akkaUid: String = context.system.asInstanceOf[ExtendedActorSystem].uid.toString
+  private def akkaUid: String = context.system.asInstanceOf[ExtendedActorSystem].uid.toString
```

Comment:
Thanks, that is a good change. However, I think there is a possibility that we log something (and thereby use this) before the uid has been generated. startDefaultLoggers is called from LocalActorRefProvider.init and that happens before RemoteActorRefProvider.init has completed. I'll take a closer look.